src/fleet/azext_fleet/tests/latest/test_fleet_scenario.py

```diff
@@ -91,6 +91,8 @@ def test_fleet(self):
         self.cmd('fleet member show -g {rg} --fleet-name {fleet_name} -n {member_name}', checks=[
             self.check('name', '{member_name}')
         ])
+
+        self.cmd('aks wait -g {rg} -n {member_name} --created', checks=[self.is_empty()])
```

Comment:
add aks wait command since instantly deleting member might leave aks in reconciliation state which causes failure while deletion.